src/modules/code-generator/base-generator.js

```diff
@@ -142,10 +143,17 @@ export default class BaseGenerator {
         value: `await ${this._frame}.waitForSelector('${selector}')`,
       })
     }
-    block.addLine({
-      type: eventsToRecord.CLICK,
-      value: `await ${this._frame}.click('${selector}')`,
-    })
+    if (!selector.startsWith('xpath:')) {
```

Comment:
Can you explain why you verify `xpath` tags?

I think we can simplify this code a little bit:
```js
const value =  selector.startsWith('xpath:') ? 
`await ${this._frame}.click('${selector}')` : 
`(await ${this._frame}.$x('${selector.substr(6)}'))[0].click()`

 block.addLine({ type: eventsToRecord.CLICK, value })
```

Reply:
I'm using xpath in my resolver because not everything is possible with css selectors alone.
I thought it would be good to separate them like url schema's: thus `xpath://*[text()="heyo"]`, where `//*[text()="heyo"]` is the xpath itself